code/game/machinery/computer/card.dm

```diff
@@ -68,7 +68,6 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 	if(isidcard(I))
 		var/obj/item/card/id/insert_card = I
 		if(!insert_card.electric)
-			to_chat(user, to_chat(user, "<span class='warning'>There's no circuitry to change!</span>"))
 			return
```

Comment:
this is not addressed because it still returns and thus won't call id_insert

Please test with modular computer card insert and see if it actually says the other message.

Otherwise, remove the return and see if it says it then.